Plots/Streamlines/NCL_stream_9.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_stream_9.py
+===============
+This script illustrates the following concepts:
+   - Showing features of the new color display model
+   - Using opacity to emphasize or subdue overlain features
```

Comment:
Is this comment true? I.e. does the plot use opacity?

Reply:
I just added a set_alpha line that changes the opacity of the plot, so this comment is applicable now. I took out the other comments.